### ISSUE TITLE
Import / export cURL for request with multipart/form-data looses content-type of parts

### ISSUE THREAD

**Issue by MosheElisha** [a85/POSTMan-Chrome-Extension#8506]
**Describe the bug**

**To Reproduce**
1. Click Import > Raw Text and paste the collowing cURL:

```
curl --location --request POST 'http://localhost:8080/api/v1/templates/' --header 'accept: application/json' --header 'Content-Type: multipart/form-data' --form 'request={ "title": "My template" };type=application/json' --form 'file=@/tmp/archive.zip;type=application/octet-stream'
```

2. Click Continue and Import

**Expected behavior**
I would expect that the content type defined for each part (application/json and application/octet-stream) will be assigned into the CONTENT TYPE column.
Instead, the CONTENT TYPE column shows "Auto" and the ";type=..." exists in the VALUE column.

After I manually edit the VALUE and CONTENT TYPE columnד, POSTMAN sends the request properly (screenshot attached).
That said, when I click Code and cURL - the CONTENT TYPE is loosed and the cURL generated is missing the content-type for the parts.

**Screenshots**
After import:
![image](https://user-images.githubusercontent.com/1391335/82339486-bf608600-99f6-11ea-8a69-435fcc345ab6.png)

Expected (after manually editing):
![image](https://user-images.githubusercontent.com/1391335/82340121-87a60e00-99f7-11ea-9aec-64a6c8e38233.png)

**App information**
 - App Type Native App
 - Postman Version v7.24.0
 - OS: Windows 10 Home

**Comment by muthu2404** [comment]
Did you find the solution for it? I'm facing the same issue.

**Comment by hugobrancowb** [comment]
same here

**Comment by VShingala** [comment]
@MosheElisha @muthu2404 @hugobrancowb This issue is now fixed with latest Postman App v9.1.2.
